docs/configuration-reference.md

```diff
@@ -1440,6 +1444,39 @@ scrape_configs:
 [target_config: <promtail.target_config>]
 ```
 
+### tempo_config
+
+The `tempo_config` block configures how the Agent receives traces and sends them to Tempo. 
+
+```yaml
+remote_write:
+  # host:port to send traces to.  Required.
+  url: <string> | default = ""
+
+  # Controls whether or not TLS is required.  See https://godoc.org/google.golang.org/grpc#WithInsecure
+  insecure: <boolean> | default = false
```

Comment:
It's GRPC so I don't think you can specify TLS through the scheme.

Reply:
You're right, but we can parse it before passing it along to OTel.

Reply:
Joe and I chatted offline and agreed that `url` with a scheme is confusing because it might appear like this is using HTTP/1 instead of gRPC (which doesn't support schemes). We agreed to go for a combo of `endpoint`/`insecure`